### ISSUE TITLE
Accessibility Issue: Images are missing alt attribute

### ISSUE THREAD

**Issue by brian101co** [zappycode/zappycode-django#47]
Adding alt attributes to the page images should help make the page more accessible to users who might be using screen readers. Also help Google know what the image is about and should help improve zappycode's onpage SEO.
